src/spec-node/devContainersSpecCLI.ts

```diff
@@ -545,6 +549,7 @@ async function doBuild({
 			skipPostAttach: true,
 			experimentalImageMetadata,
 			skipPersistingCustomizationsFromFeatures: skipPersistingCustomizationsFromFeatures,
+			skipPersistingRemoteEnvFromConfig: false,
```

Comment:
Why hardcode this here and in other places instead of using `skipPersistingRemoteEnvFromConfig` from the command line arg? It seems like if you build an image separately, you'd still want this flag honored.

Reply:
From the Codespaces perspective we only create containers with `up` command, so I have added the commandline option only for the `up` command. But the downstream code is shared by most commands and the downstream code expects this param.
A simple alternative would have been to make this flag optional `skipPersistingRemoteEnvFromConfig? : boolean` in the downstream, so that we don't have to define them with `false` value everywhere else. I was just following the pattern @joshspicer established for `skipPersistingCustomizationsFromFeatures`.

Reply:
On a second thought, I felt it is better to cleanup the code and avoid the need for setting default value in other commands.
Done